scripts/add_electricity.py

```diff
@@ -229,69 +261,78 @@ def attach_generators(n, costs, generators=None, build_years=None, model_year=No
                     name = f"{bus} {tech} {model_year}"
                     extendable = tech in snakemake.config['extendable_techs']
                     build_year = model_year
-                    
+
                 # ramp limits
                 if tech in ['LWR']:
-                    ramp_limit_up = 0.01*resolution
-                    ramp_limit_down = 0.01*resolution
+                    ramp_limit_up = 0.01 * resolution
+                    ramp_limit_down = 0.01 * resolution
                     p_nom_min = p_nom
                 elif tech in ['IGCCAvgCF', 'Biopower']:
-                    ramp_limit_up = 0.1*resolution
-                    ramp_limit_down = 0.1*resolution
+                    ramp_limit_up = 0.1 * resolution
+                    ramp_limit_down = 0.1 * resolution
                     p_nom_min = 0
-                elif tech in ['CCAvgCF','CTAvgCF']:
-                    ramp_limit_up = min(0.6*resolution,1.0)
-                    ramp_limit_down = min(0.6*resolution,1.0)
+                elif tech in ['CCAvgCF', 'CTAvgCF']:
+                    ramp_limit_up = min(0.6 * resolution, 1.0)
+                    ramp_limit_down = min(0.6 * resolution, 1.0)
                     p_nom_min = 0
                 elif tech in ['Petroleum']:
                     ramp_limit_up = 1.0
                     ramp_limit_down = 1.0
                     p_nom_min = 0
-                else:   
+                else:
                     ramp_limit_up = 1.0
                     ramp_limit_down = 1.0
                     p_nom_min = p_nom
 
                 # minimum/maximum power output
                 if tech == 'LWR':
-                    p_min_pu = 0.95
-                    p_max_pu = 1.0  
+                    p_min_pu = 0
```

Comment:
Just a note that would be nice to get these numeric values (and any others) moved over to the config file in a future PR.